src/spikeinterface/widgets/unit_summary.py

```diff
@@ -136,37 +140,41 @@ def plot_matplotlib(self, data_plot, **backend_kwargs):
             ax1.set_xlabel(None)
             ax1.set_ylabel(None)
 
-        ax2 = fig.add_subplot(gs[:2, 1])
+        ax2 = fig.add_subplot(gs[:2, col_counter])
         w = UnitWaveformsWidget(
             sorting_analyzer,
             unit_ids=[unit_id],
             unit_colors=unit_colors,
             plot_templates=True,
+            plot_waveforms=sorting_analyzer.has_extension("waveforms"),
```

Comment:
If `UnitWaveformsWidget` is the only plot that is always plot, could it be moved to the top of this section? This would help make it clearer what plots are conditional and what plots are always plot (i.e. always plot at the top, then conditional).

Reply:
Actually I guess this doesn't work if this plot should be after the first `UnitWaveformsWidget` (if it exists). Maybe a comment above like `#This plot is always displayed, if 'unit_locations' exists it will be plot in the column before it`.